sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -64,15 +62,12 @@ public IReadOnlyList<GeneratorFunctionMetadata> GetFunctionMetadataInfo(List<Met
                         ScriptFile = scriptFile
                     };
 
-                    // collect Bindings
-                    var bindings = GetBindings(method, model, functionName, out bool hasBindingError, out bool hasHttpTrigger);
-
-                    if(hasBindingError)
+                    if (!TryGetBindings(method, model, out IList<IDictionary<string, string>>? bindings, out bool hasHttpTrigger) || bindings is null)
```

Comment:
No that case won't work since all Functions need to have at least one binding - if no bindings are returned, then the function is invalid. I'll remove the `|| binding is null` check then.